packages/docs/src/page-configs/ui-elements/dropdown/examples/PlacementAndOffset.vue

```diff
@@ -43,17 +43,23 @@
 </template>
 
 <script>
+import { aliasToPlacement } from 'packages/ui/src/composables'
```

Comment:
This bailed out a warning: Can't resolve 'packages/ui/src/composables'. How can I solve this issue?

Reply:
1. Proper composable name is `usePlacementAliases`.
2. You shouldn't try to import it directly - it should be used only via `VaDropdown` component and props that affect this functionality.

Reply:
Oh, sorry, `composables` are already included.